### PR TITLE
Added configuration options and local server

### PR DESCRIPTION
**Pull request by matt-oconnell**
Added Firefox configuration options and local server to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,10 @@ This is a prototype debugger written without any XUL and based on React and Redu
 
 #### Getting Started
 
-```js
+```
 $ npm install
 $ npm start
+$ python -m SimpleHTTPServer 8000
```
Comment (jasonLaster) [inline README.md]: This is no longer necessary: [npm start](https://github.com/jlongster/debugger.html/blob/master/package.json#L4)

File: README.md [hunk 2]
```diff
@@ -17,10 +18,17 @@ Start Firefox in remote debugging mode and go to a tab you want to debug.
...
 
-Go to the Debugger!
+Set additional configuration options in Firefox using `about:config`:
+
+- `devtools.debugger.remote-enabled` to `true`
+- `devtools.chrome.enabled` to `true`
+- `devtools.debugger.prompt-connection` to `false`
+
+
+Open the debugger in another browser by visiting
```
Comment (jasonLaster) [inline README.md]: thanks, this was something that a lot of people needed yesterday

File: README.md [hunk 3]
```diff
@@ -17,10 +18,17 @@ Start Firefox in remote debugging mode and go to a tab you want to debug.
 $ /Applications/FirefoxNightly.app/Contents/MacOS/firefox-bin --start-debugger-server 6080
 ```
 
-Go to the Debugger!
+Set additional configuration options in Firefox using `about:config`:
+
+- `devtools.debugger.remote-enabled` to `true`
+- `devtools.chrome.enabled` to `true`
+- `devtools.debugger.prompt-connection` to `false`
```
Comment (jasonLaster) [inline README.md]: thanks, this was something that a lot of people needed yesterday

Comment (jasonLaster) [inline README.md]: afterwards, maybe say. Quit and re-open firefox with the same command.

**Comment by jasonLaster** [comment]
as a sidenote, it might be nice to encourage people to use a profile for the firefox browser, this would make it easier to keep settings, tabs, and have multiple firefox instances open at the same time. Perhaps this is in the Advanced section, so that it's not another step early on?

**Comment by matt-oconnell** [comment]
@jasonLaster I'm unfamiliar with Firefox profiles. Do you mean configuring it through `/Applications/Firefox.app/Contents/MacOS/firefox-bin -P`?

**Comment by jasonLaster** [comment]
@matt-oconnell exactly.

**Comment by jasonLaster** [comment]
@jlongster what do you think?

File: README.md (hunk 3 above)
Comment (jlongster) [inline README.md]: Why do we need `devtools.chrome.enabled` ? This is good to document, definitely

**Comment by jlongster** [comment]
Sounds good! Is just adding `-P development` to the command to run Firefox enough? That's what I do.

**Comment by jlongster** [comment]
We can go ahead and merge this though and just keep tweaking it

**Comment by jlongster** [comment]
I'll go ahead and merge this while I'm here. I'll go ahead and add a note about the `-P` parameter too.

**Pull request merged**